dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -414,4 +424,50 @@ public Config getConfig() {
         }
     }
 
+    @GET
+    @Timed
+    @Path("cli-info")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get Dockstore CLI information.", description = "Get Dockstore CLI information. NO authentication")
+    @ApiOperation(value = "Get Dockstore CLI information.", notes = "Get Dockstore CLI information. NO authentication")
+    public CLIInfo getCliVersion() {
+        GHRelease ghRelease;
+        PagedIterable<GHRelease> allReleases;
+        CLIInfo cliInfo = new CLIInfo();
+        try {
+            GitHub github = new GitHubBuilder().build();
+            GHRepository repository = github.getRepository("dockstore/dockstore-cli");
+            ghRelease = repository.getLatestRelease();
+            allReleases = repository.listReleases();
+        } catch (IOException e) {
+            LOG.info("Could not get CLI releases information from GitHub. ", e);
+            throw new CustomWebApplicationException("Could not get CLI  releases information info from GitHub",
+                    HttpStatus.SC_SERVICE_UNAVAILABLE);
+        }
+        String cliLatestVersion = ghRelease.getName();
+        cliInfo.setCliLatestDockstoreScriptDownloadUrl(DOCKSTORE_CLI_RELEASES_URL + "/download/" + cliLatestVersion + "/dockstore");
+        cliInfo.setCliLatestVersion(cliLatestVersion);
+        // Get the latest unstable version
+        // If there is no unstable version return an empty string as the latest unstable version
```

Comment:
I know this should never happen because we have unstable versions and they're not going away, but if there is no unstable version, but I don't think we should have an empty string and a non-working url if we can't find one. Should probably set both values to null.

Reply:
Now returns null